references/contributing/style_guide.md

```diff
@@ -313,7 +313,7 @@ test myTestFunction ... ok
 Example of test:
 
 ```ts, ignore
-import { assertEquals } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts";
+import { assertEquals } from "https://deno.land/std@$STD_VERSION/assert/assert_equals.ts";
```

Comment:
Can you change all these imports to use `assert/mod.ts` instead?

Reply:
PR is updated to use `assert/mod.ts`